app/helpers/application_helper.rb

```diff
@@ -409,6 +409,10 @@ def edit_inline(object, property, options = {})
     end
   end
 
+  def foreman_doc_url(text = "")
```

Comment:
There's already `documentation_button`, I'd suggest this shouldn't have a "foreman_" prefix to match and should be unified.